src/main/scala/io/citrine/lolo/Learner.scala

```diff
@@ -1,27 +1,32 @@
 package io.citrine.lolo
 
-/**
-  * Created by maxhutch on 11/14/16.
-  */
+import io.citrine.random.Random
+
 trait Learner extends Serializable {
 
   /**
     * Train a model
     *
     * @param trainingData to train on
     * @param weights      for the training rows, if applicable
+    * @param rng          random number generator for reproducibility
     * @return training result containing a model
     */
-  def train(trainingData: Seq[(Vector[Any], Any)], weights: Option[Seq[Double]] = None): TrainingResult
+  def train(
+      trainingData: Seq[(Vector[Any], Any)],
+      weights: Option[Seq[Double]] = None,
+      rng: Random = Random()
```

Comment:
Do we want to use the `Random.default` method?

Reply:
Edit: I forget why we even have `Random.default` vs `Random()`, both of which instantiate an effectively unseeded `Random`. I guess one was done before the other.

Reply:
Different idioms for doing the same thing 🤷